src/help.jl

```diff
@@ -4,10 +4,41 @@ function GAP_help_string(topic::String, onlyexact::Bool = false)
     return GAP.gap_to_julia(GAP.Globals.HelpString(GAP.julia_to_gap(topic), onlyexact))
 end
 
+"""
+    show_GAP_help(topic::String, onlyexact::Bool = false)
```

Comment:
Hmm, elsewhere we use `gap` in lowercase in function names, and that also seems to be more the "Julia way", so perhaps this function should also do that?
```suggestion
    show_gap_help(topic::String, onlyexact::Bool = false)
```